### ISSUE TITLE
[competition] Compiler validation with benchmark sources

### ISSUE THREAD

**Issue by hyn2028** [kaist-cp/cs420#417]
Hello.

Are all benchmark sources for competition open and available for testing by the compiler?

My compiler has been verified to work only against the test source used to pass the assignment.

That is, when compiling a new source, a todo assertion may occur or incorrect compilation may occur.

Is it possible to test with the source to be used in advance before submitting to the competition?

Thank you.

**Comment by minseongg** [comment]
Yes, the benchmark codes are defined in [here](https://github.com/kaist-cp/kecc-public/tree/main/bench) and you can run the benchmark by `make run`.

**Comment by hyn2028** [comment]
Thank you!!
